news/_posts/starter-theme/2015-12-17-starter-theme-news-3.markdown

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title:  "Starter Theme: fresh news from the front #3"
+subtitle: "What's up about Starter Theme"
+date:   2015-12-17 15:00:00
+author:  maximebiloe
+icon: icon-lego
+published: false
+---
+
+It's been a while since we gave you any news about the StarterTheme. So, let's sum up what happened since [Starter Theme: Fresh News From The Front #2](http://build.prestashop.com/news/starter-theme-news-2/).
+
+## Page layout
+
+We don't want to impose a 3 columns layout for template developers anymore. We want to make something more flexible. As a theme developer, you will be able to ship your theme with as many layouts as you want. For example you can have a full width layout, 2 columns split in 2 halves or one third/two third. Better yet, you can add hooks inside template so you can have specific sidebars. PrestaShop users will be able to select the wanted layout directly in the back office, among those supported by the theme.
+
+## Checkout process
+
+We have removed the 5-steps checkout process. From now on, there will be the **One-Page-Checkout only**. Template designers will be able to easily transform it in other types of checkout by playing with JavaScript.
+
+Another big change regarding the checkout process is that it will be displayed with a **simplified and minimalistic layout**, without any website decoration, in order to prevent the customers from being distracted during the checkout process.
+
+By default, **guest checkout will be enabled**, so the password will be facultative. Customers will be re-invited to register at the end of the process. This way, we expect the purchase process to be much quicker and more effective.
+
+The Starter Theme also displays a cart summary, without any actions possible, on the right. It allows the customer to always know what he is buying, how much it will cost him, etc.
+
+![PrestaShop New Checkout Process](/assets/images/2015/12/new_checkout_process.png)
+
+## Payment API
+
+To provide the best user experience possible for customers, and to guarantee that PrestaShop orders respect the law in all countries, we implemented a new payment API. To do that, we chose to respect the strictest requirements we know about : the German ones.
```

Comment:
`the strictest requirements we know about` => maybe elaborate some more, because I think this is a bit scary. e.g. "The German requirements being strict doesn't mean that the checkout process will be tedious. To the contrary, the way we need to structure the page to abide by German best practices results in a user experience that is very reassuring to customers and helps increase conversion rate."